1Password/op-scim-gcp-marketplace
# Fix version and public IP

This PR resolves the issue where the public IP address was no longer being obtained from the SCIM bridge service, and displayed in the SCIM bridge info section.

The root cause of the issue is that with the migration to using the `1Password/op-scim-helm` Helm chart we neglected to update the reference to the service name.

To test:

1. Ensure your cluster doesn't have any existing applications: `kubectl delete application --all`
2. Wait for the resources to be cleaned up, including the SCIM bridge load balancer service
3. From the `op-scim-bridge` folder (where the `Makefile` is), build and tag a test image:
     ```
     REGISTRY="gcr.io/op-scim-bridge" TAG=test-fix-for-54 make app/build
     ```
4. Install the test tag with mpdev:
    ```
    mpdev install --deployer=gcr.io/op-scim-bridge/op-scim-bridge/deployer:test-fix-for-54 --parameters='{"name": "op-scim-bridge-1", "namespace": "op-scim-bridge", "accountDomain": "example.1password.com" }'
    ```
5. Ensure the application installs correctly and that the deployer creates the correct resources.
6. Ensure that the SCIM bridge service public IP address is displayed under the "SCIM bridge info" section.
7. Ensure that clicking the IP address opens the deployed SCIM bridge UI.

Resolves #54.
